### ISSUE TITLE
13Z055-90 cleanup

### ISSUE THREAD

**Issue by dpfeuffer** [MEN-Mikro-Elektronik/13MD05-90#271]
Sorry for the late issue assigned to 02_05 release, but we just discovered some important work to do:
1. There is an old dangling branch from me (dp_src_update) that wasn't considered in the past and contains partly changes that are now added again in a new pull request.
2. There is another dangling branch (mad_dev - underscore, not '-') which solves a customer problem.
3. There is a new pull request: https://github.com/MEN-Mikro-Elektronik/13Z055-90/pull/9

Please cleanup the https://github.com/MEN-Mikro-Elektronik/13Z055-90 repo:
- merge the open pull request from branch feat/rename_nrzi-s into mad-dev
- investigate if there are unconsidered changes in dp_src_update which should be merged into mad-dev
- investigate if there are unconsidered changes in mad_dev (underscore, not '-')  which should be merged into mad-dev
- delete all branches which are no longer required

**Comment by mad-jsanjuan** [comment]
OK. So the problem with that `dp_src_update` branch is that the pull request created was not merged properly. The pull request got closed and the changes were applied manually instead.

The original closed PR:

https://github.com/MEN-Mikro-Elektronik/13Z055-90/pull/1

The two commits that "merged" the changes instead of merging the original branch:

https://github.com/MEN-Mikro-Elektronik/13Z055-90/commit/96f553af2d6a64608d41c225e98201f50f2b3fa0
https://github.com/MEN-Mikro-Elektronik/13Z055-90/commit/6150ddf03861f9010ef2fe58f6ff2aabd9ed668d

The changes are there so it is safe to keep the `dp_src_update` branch deleted.

**Comment by mad-jsanjuan** [comment]
With regards to that `mad_dev` branch, this branch was not properly created and it mixes some not finished commits with the actual fix that solves MAIN_PR007849. The actual fix got directly merged into master with this PR: https://github.com/MEN-Mikro-Elektronik/13Z055-90/pull/4 . We then created a new `mad-dev` branch based on master.

We can remove that confusing `mad_dev` branch as we faced some issues with the mix of approaches there for fixing MAIN_PR007849 instead of just the final solution merged to master.

**Comment by dpfeuffer** [comment]
I'm confused that the dp_src_update and mad_dev branch is still shown in the network view of the repo:
![image](https://user-images.githubusercontent.com/31008280/215424523-4bccf97a-d510-45bf-b044-80f904f59a00.png)
Also confusing: Why is "gvarlet" assigned to my dp_.. branch in the row? 

Whereas the branch overview shows only the master and mad-dev branch:
![image](https://user-images.githubusercontent.com/31008280/215424886-ec09db5b-e332-4382-b76f-74a3b7c03b84.png)

So maybe dp_src_update and mad_dev where already deleted as I wrote this issue? 
In the past I deleted a branch in a repository and it disappeared from the network tree - or was it in GitLab? Don't know. ;-)
